www/src/components/shell/onboarding/OnboardingHeader.tsx

```diff
@@ -0,0 +1,86 @@
+import { useTheme } from 'styled-components'
+import { Dispatch } from 'react'
+import {
+  Button,
+  DiscordIcon,
+  DocumentIcon,
+  IconFrame,
+  LifePreserverIcon,
+  Sidebar,
+  SidebarItem,
+  SidebarSection,
+} from '@pluralsh/design-system'
+
+import { OnboardingLogo } from './OnboardingLogo'
+
+interface OnboardingHeaderProps {
+  onRestart?: Dispatch<void>
+}
+
+function OnboardingHeader({ onRestart }: OnboardingHeaderProps) {
+  const theme = useTheme()
+
+  return (
+    <Sidebar
+      layout="horizontal"
+      background={theme.colors['fill-one']}
```

Comment:
Using just `backgroundColor="fill-one"` doesn't work?

Reply:
This is reused from the old code. I didn't change that part, but I assume it is required.